js/components/glossary.js

```diff
@@ -1,7 +1,6 @@
 ---
+# Jekyll front matter lives here.
```

Comment:
Expected an identifier and instead saw '#'.<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.<br>'Jekyll' is not defined.<br>'front' is not defined.<br>'matter' is not defined.<br>'lives' is not defined.<br>'here' is not defined.